packages/components/cascader-panel/src/node-content.ts

```diff
@@ -2,6 +2,13 @@
 import { defineComponent, h } from 'vue'
 import { useNamespace } from '@element-plus/hooks'
 
+import type { VNode } from 'vue'
+
+//https://github.com/vuejs/core/issues/4733#issuecomment-933284261
+function isVNodeEmpty(vnodes?: VNode[]) {
+  return !!vnodes?.every((vnode) => vnode.type !== Comment)
```

Comment:
```suggestion
  return !!vnodes?.every((vnode) => vnode.type === Comment)
```
Here, as long as the slot content is not all comments, the slot content should be rendered.

`Comment` does not seem to be introduced, and the content judgment method can be called.
https://github.com/element-plus/element-plus/blob/1c3fc40a3c4a8f592306005834995d1eebb7e1d6/packages/utils/vue/vnode.ts#L59-L61